lyft_dataset_sdk/eval/detection/mAP_evaluation.py

```diff
@@ -126,7 +126,7 @@ def calculate_ground_bbox_coords(self):
         rotation_matrix = self.quaternion.rotation_matrix
 
         cos_angle = rotation_matrix[0, 0]
-        sin_angle = rotation_matrix[0, 1]
+        sin_angle = rotation_matrix[1, 0]
```

Comment:
Looks like a pretty big change, I'm wondering how it worked before?
Also, Looks like it should be negated now, because rotation matrix is usually:
```
cos, sin
-sin, cos
```

Reply:
Rotation matrix is

```
cos, -sin
sin, cos
``` 
https://en.wikipedia.org/wiki/Rotation_matrix